contracts/exchange/test/core.ts

```diff
@@ -402,6 +405,17 @@ describe('Exchange core', () => {
                 (transferLogs[1] as LogWithDecodedArgs<DummyERC20TokenTransferEventArgs>).args._value,
             ).to.be.bignumber.equal(signedOrder.takerFee);
         });
+
+        it('should throw if order is expired', async () => {
```

Comment:
Any reason this was added? I think it's pretty well tested in `fill_order.ts`. Maybe some of these tests should be restructured at some point so that this is more obvious.

Reply:
Oh, I disabled the expired orders in the combinatorial tests to make up for fee tokens exponentially introducing more tests. I didn't see a benefit in testing expiration combinations since that logic throws well ahead of any accounting. So I added this test to make up for it.